docs/monitoring/logging.md

```diff
@@ -93,7 +93,7 @@ Placeholders can also be used in conjunction with exceptions which shall be logg
 
 {% highlight java %}
 catch(Exception exception){
-	LOG.error("An {} occurred.", "error", exception);
```

Comment:
I do not think this is needed. After the change, there will be no stack trace.

Reply:
@KarmaGYZ I think "LOG.error("An {} occurred", "error", exception)" is intend, just as the documentation said above "Placeholders can also be used in conjunction with exceptions which shall be logged."

Reply:
Do slf4j has `void error(String format, Object arg, Throwable t)` API? 
I couldn't find it in 1.7.15 and [slf4j-1.8.0](https://www.slf4j.org/api/org/slf4j/Logger.html).

Reply:
Why the third parameter has to be `Throwable`? Maybe you could have a try in IDEA :)

Reply:
@klion26 you're right, thanks for the review!
Close it.